Packs/Mattermost/Integrations/MattermostV2/MattermostV2.yml

```diff
@@ -521,12 +538,121 @@ script:
+      description: How many results to retrieve. Will override the page and page_size
+        arguments if given.
+    - name: group
+      description: Search for a specific group by this pattern.
+    name: mattermost-list-groups
+    outputs:
+    - contextPath: Mattermost.Groups.id
+      description: The ID of the group.
+      type: String
+    - contextPath: Mattermost.Groups.create_at
+      description: When was the group created.
+      type: Unknown
+    - contextPath: Mattermost.Groups.update_at
+      description: When was the group updated.
+      type: Unknown
+    - contextPath: Mattermost.Groups.delete_at
+      description: When was the group deleted.
+      type: Unknown
+    - contextPath: Mattermost.Groups.display_name
+      description: The display name of the group.
+      type: String
+    - contextPath: Mattermost.Groups.name
+      description: The name of the group.
+      type: String
+    - contextPath: Mattermost.Groups.description
+      description: The description of the group.
+      type: String
+    - contextPath: Mattermost.Groups.source
+      description: The source of the group.
+      type: String
+    - contextPath: Mattermost.Groups.remote_id
+      description: The remote id of the group.
+      type: String
+    - contextPath: Mattermost.Groups.has_syncables
+      description: If the group has any syncables.
+      type: boolean
+    description: 'Lists groups. Groups are available on Enterprise and Professional plans. Minimum server version: 5.11'
+  - name: mattermost-list-group-members
+    arguments:
+    - name: group_id
+      required: true
+      description: The group id of the group to list members for. Use the command
+        'mattermost-list-groups' to fetch the group ID.
+    - name: page
+      description: The page number to retrieve. Default value is 0.
+    - name: page_size
+      description: The size of the page to retrieve. Default value is 50.
+    - name: limit
+      description: How many results to retrieve. Will override the page and page_size
+        arguments if given.
+    outputs:
+    - contextPath: Mattermost.Groups.id
+      description: The ID of the group.
+      type: String
+    - contextPath: Mattermost.Groups.total_member_count
+      description: The total count of members in the group.
+      type: Unknown
+    - contextPath: Mattermost.Groups.members
+      description: Detailed list of the group members.
+      type: Unknown
+    description: 'Lists group members. Groups are available on Enterprise and Professional plans. Minimum server version: 5.11'
+  - name: mattermost-add-group-member
+    arguments:
+    - name: group_id
+      required: true
+      description: The group id of the group to add the user to. Use the command
+        'mattermost-list-groups' to fetch the group ID.
+    - name: user_ids
+      required: true
+      isArray: true
+      description: The ID(s) of the user(s) to add. Use the command 'mattermost-list-users'
```

Comment:
Is this a comma-separated list?
If so, the description should be: A comma-separated list of IDs of the users to add......